sympy/solvers/ode/systems.py

```diff
@@ -1236,6 +1236,28 @@ def _component_solver(eqs, funcs, t):
     return sol
 
 
+def _higher_order_to_first_order(eqs, sys_order, t):
+    new_funcs = []
+
+    for func in sys_order:
+        new_funcs.append(func)
+        subs_dict = {func: func}
+        new_eqs = []
+
+        for i in range(1, sys_order[func]):
+            new_func = Function(Dummy())(t)
```

Comment:
We should give these dummies names so e.g. `x(t).diff(t)` would be `Function(Dummy('x_1'))(t)` and `x(t)` would be `Function(Dummy('x_0')(t))`.

Reply:
I would do that for functions introduced but should I do it for dependent variables like `x(t)` since we need to access the solutions later using a dictionary and the original dependent variables don't really have to be renamed.